apps/ejabberd/src/mod_mam_odbc_async_pool_writer.erl

```diff
@@ -60,6 +60,13 @@ worker_prefix() ->
 worker_count(_Host) ->
```

Comment:
It is actually good, idea. I will pull request soon with rebased commits without changes, and new implementation.